core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/ImageCatalogV4Controller.java

```diff
@@ -188,7 +189,7 @@ public ImageV4Response getSingleImageByCatalogNameAndImageId(Long workspaceId, @
 
     @Override
     @AccountIdNotNeeded
-    @DisableCheckPermissions
+    @InternalOnly
     public ImageV4Response getImageFromDefaultById(Long workspaceId, @ResourceName String imageId) throws Exception {
 
         // FIXME: Implement this!
```

Comment:
Wohoo, I know this is not related to this ticket but shouldn't we have a ticket to implement this?

Reply:
https://jira.cloudera.com/browse/CB-12624

Reply:
This endpoint is going to be called from CDB CLI. Will InternalOnly annotation allows CDP CLI to call it?